setup.cfg

```diff
@@ -0,0 +1,17 @@
+[isort]
+known_first_party=sqlalchemy_utils,tests
+line_length=88
+multi_line_output=3
+order_by_type=false
+include_trailing_comma=True
+
+[flake8]
+ignore = D10,E203,E501,W503,E231,E265,W504,E122,E121
```

Comment:
Is there a reason for adding all these ignores?

Reply:
Some of these were already set, and there were already a lot of violation for flake8 so for the most common ones I added it to ignore. If I remove them then I would have to do a lot of editing to code which I am not too sure if you would like it.

Reply:
@kvesteri could you please confirm if these ignores are ok? I can then submit my PR. Thanks